lib/src/HttpAppFrameworkImpl.cc

```diff
@@ -58,9 +58,21 @@
 #include <sys/file.h>
 #include <uuid.h>
 #include <unistd.h>
+#define os_access access
 #else
 #include <io.h>
+#define os_access _waccess
+#define R_OK 04
+#define W_OK 02
 #endif
+// Switch between native c++17 or boost for c++14
+#ifdef USE_BOOST_FILESYSTEM
+#include <boost/filesystem.hpp>
+namespace fs = boost::filesystem;
+#else   // USE_BOOST_FILESYSTEM
+#include <filesystem>
+namespace fs = std::filesystem;
+#endif  // USE_BOOST_FILESYSTEM
```

Comment:
Please pack this into `drogon/utils/filesystem.h` like how `any` and `string_view` does

Reply:
And put the `boost/std::filesystem` namespace into `drogon::fs`

Reply:
Yes, that's what I wanted to do at the next step. But I can also do that right now.